database/migrations/008_optimistic.go

```diff
@@ -0,0 +1,53 @@
+package migrations
+
+import (
+	"github.com/flashbots/mev-boost-relay/database/vars"
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+var Migration008Optimistic = &migrate.Migration{
+	Id: "008-optimistic",
+	Up: []string{
+		`
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD optimistic_submission bool NOT NULL default false;
```

Comment:
Do we want any additional indexes on any of these? (i.e on `optimistic_submission`)?

Reply:
i think we are ok honestly! we haven't faced any issues when querying the db manually. for sure open to it in the future, but as is, i dont think it is necessary!